src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,225 @@
+import * as React from 'react';
+import { ColorPaletteProp } from '@mui/joy/styles';
+import AspectRatio from '@mui/joy/AspectRatio';
+import Box from '@mui/joy/Box';
+import Button from '@mui/joy/Button';
+import IconButton from '@mui/joy/IconButton';
+import Card from '@mui/joy/Card';
+import CardContent from '@mui/joy/CardContent';
+import Chip from '@mui/joy/Chip';
+import Divider from '@mui/joy/Divider';
+import Input from '@mui/joy/Input';
+import List from '@mui/joy/List';
+import ListSubheader from '@mui/joy/ListSubheader';
+import ListItem from '@mui/joy/ListItem';
+import ListItemDecorator from '@mui/joy/ListItemDecorator';
+import ListItemButton from '@mui/joy/ListItemButton';
+import Typography from '@mui/joy/Typography';
+import Sheet from '@mui/joy/Sheet';
+import FacebookRoundedIcon from '@mui/icons-material/FacebookRounded';
+import GitHubIcon from '@mui/icons-material/GitHub';
+import SendIcon from '@mui/icons-material/Send';
+
+export default function Footer() {
+  const [color, setColor] = React.useState<ColorPaletteProp>('neutral');
+  return (
+    <Sheet
+      variant="solid"
+      color={color}
+      invertedColors
+      sx={{
+        ...(color !== 'warning' && {
+          bgcolor: `${color}.800`,
+        }),
+        flexGrow: 1,
+        p: 2,
+        mx: -3,
+        my: -3,
+        borderRadius: { xs: 0, sm: 'xs' },
+      }}
+    >
+      <Box sx={{ display: 'flex', alignItems: 'center', gap: 2 }}>
+        <IconButton
+          variant="soft"
+          size="sm"
+          onClick={() => {
+            const colors: ColorPaletteProp[] = [
+              'primary',
+              'neutral',
+              'danger',
+              'info',
+              'success',
+              'warning',
+            ];
```

Comment:
looks like constant to export

Reply:
I changed from mui/joy to mui library and concept. I have added the changes for approval.